MedvedevKS/lab_2/lab_2.cpp

```diff
@@ -0,0 +1,60 @@
+#include "stdafx.h"
+#include "iostream"
+#include "conio.h"
+using namespace std;
+
+
+int main()
+{
+	const int size(11);
+	const int size1 = size / 2 + size % 2;
+	int random_arr[size], odd_arr[size1];
+	cout << "Generated 1st array: \t\t";
+	int i(0);
+	for (i; i < size; i++)
+	{
+		random_arr[i] = rand();
+		cout << random_arr[i] << " ";
+	}
+	cout << endl;
+	i = 0;
+	int i1(0);
+	int changer;
+	cout << "Sorted 1st array: \t\t";
+	while (i < size)
+	{
+		i1 = i;
+		while (i1 < size)
+		{
+			if (random_arr[i1] < random_arr[i])
+			{
+				changer = random_arr[i1];
+				random_arr[i1] = random_arr[i];
+				random_arr[i] = changer;
+			}
+			i1++;
+		}
+		cout << random_arr[i] << " ";
+		i++;
+	}
+	cout << endl;
+	i = 0;
+	i1 = 0;
+	int sum_of_odd_arr(0);
+	cout << "2nd array: \t\t\t";
+	while (i < size1)
+	{
+		odd_arr[i] = random_arr[i1];
+		sum_of_odd_arr = sum_of_odd_arr + odd_arr[i];
+		cout << odd_arr[i] << " ";
+		i1 = i1 + 2;
```

Comment:
Second array have to contains all odd elements from first array, not elements with odd INDEXES